pages/setup/helper-tools/colour-contrast-analyser/README.md

```diff
@@ -18,8 +18,8 @@ As it does not have an installer, you can simply extract it to a folder of your
 
 ## Usage
 
-![Colour Contrast Analyser window](_media/colour-contrast-analyser-window.png)
+Simply enter a foreground and a background colour into the respective fields and the CCA returns whether or not the contrast ratio is sufficient, depending on the kind of visual content and size. The evaluation uses the recommendations from the Web Content Accessibility Guidelines for [colour contrast level AA](https://www.w3.org/TR/WCAG21/#contrast-minimum) and [colour contrast level AAA](https://www.w3.org/TR/WCAG21/#contrast-enhanced).
 
-Enter foreground and background colours into the respective fields, then check the displayed results.
+![Colour Contrast Analyser window](_media/colour-contrast-analyser-window.png)
 
-Note: if you choose to use the colour pickers instead, be sure to avoid anti-aliased colours, as this would have an impact on the result.
\ No newline at end of file
+For more information about colour contrast in general, there is a [dedicated chapter: Colors and contrast](/knowledge/colors-and-contrast/).
```

Comment:
Forward link: `If you are really curious and want to learn more about this, skip ahead and read [Colors and contrast](/knowledge/colors-and-contrast/).`